app/routes/index.js

```diff
@@ -1,12 +1,28 @@
 'use strict';
 
+const SpotifyWebApi = require('spotify-web-api-node');
 const express = require('express');
+// const spotifyApi = require('../modules/spotify');
 
 const router = express.Router();
 
-/* GET home page. */
+// TODO sacar esto a un modulo que no se porque no pude
```

Comment:
TODO = to-do. Si lo pones asi es una palabra reservada que despues la podes buscar para encontrar todo lo que falta hacer o quedo pendiente